packages/sync-repo-settings/required-checks.json

```diff
@@ -0,0 +1,154 @@
+{
+  "java": {
+    "enableSquashMerge": true,
+    "enableRebaseMerge": false,
+    "requireUpToDateBranch": false,
+    "requiredStatusChecks": [
+      "Kokoro - Test: Binary Compatibility",
+      "Kokoro - Test: Code Format",
```

Comment:
I assume we'll delete this functionality from `sloth` once we land this :+1:

Reply:
of course :)